app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -42,6 +42,7 @@ def setup_form_object(step)
           begin
             form_name.constantize
           rescue NameError => x
```

Comment:
A couple of times during dev I have had to comment out the error handler in `setup_form_object` as it swallows up the underlying error. I'd be happier to lose it altogether if possible. If we are going to resolve the form class both dynamically and via a hash map (which we are only doing because we cannot agree :smile:) then how about we delete `setup_form_object` and change `form_object_for` to
```
def form_object_for(step, enrollment)
  klass = form_object_class_map[step.to_sym]
  klass ||= "FloodRiskEngine::Steps::#{step.to_s.classify}Form". constantize
  klass.factory(enrollment)
end
```
Its just as clear what is happening (though a future developer is still going to be scratching their heads as to why we are trying both lookup methods!), and the error raised if the form does not exists will be `#<NameError: uninitialized constant FloodRiskEngine::Steps::BlahStepNonsenseForm>` which is all a developer needs to see what's happening.